unfetter-discover-api/api/controllers/x_unfetter_assessments.js

```diff
@@ -214,27 +214,30 @@ const riskPerKillChain = controller.getByIdCb((err, result, req, res, id) => { /
                 assessment = assessment.toObject().stix;
             }
 
-            const indicators = results[0]
+            const courseOfActions = results[0]
                 .filter(doc => doc !== undefined)
                 .map(doc => ({
                     ...doc.toObject().stix,
                     ...doc.toObject().metaProperties
                 }));
-            const indicatorRisks = [];
-            const sensors = results[1]
+            const coaRisks = [];
```

Comment:
Based on previous question and other `Risks` variable names below, maybe `courseRisks` would be better?